pkg/proxy/apis/config/doc.go

```diff
@@ -16,4 +16,4 @@ limitations under the License.
 
 // +k8s:deepcopy-gen=package
 
-package kubeproxyconfig // import "k8s.io/kubernetes/pkg/proxy/apis/kubeproxyconfig"
+package config // import "k8s.io/kubernetes/pkg/proxy/apis/config"
```

Comment:
this should probably be done already in the first commit, as well as moving OWNERS and BUILD I think